lib/src/excel.dart

```diff
@@ -19,6 +19,13 @@ class ExcelDart {
   static const _horizontalAlignment = 'Center';
   static const _borderStyle = 'Continuous';
   static const _currencyFormat = '\$#,##0.00';
+  static const _badFont = '#9C0006';
+  static const _badFill = '#FFC7CE';
```

Comment:
how were these values chosen? what color is this?

Reply:
This mimics the 'bad' format in Excel.
![Excel-bad-format](https://user-images.githubusercontent.com/66260095/87355009-9a3b5000-c52d-11ea-920d-532d88c1e39a.png)

Reply:
oh true. drop a comment to that effect!